.github/workflows/update-github-io.yml

```diff
@@ -9,9 +9,18 @@ on:
       version:
         description: 'Target Version without (-SNAPSHOT)'
         required: true
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        required: true
+      concurrency-group:
+        type: string
+        required: true
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  group: ${{ github.workflow }}
+    -${{ github.event.pull_request.number || github.ref }}-${{ inputs.concurrency-group }}
```

Comment:
sounds like not related optimization, move to separate PR to merge sooner.

Reply:
It is related to the PR, we use `${{ inpuits.concurrency-group }}` which we get from the caller workflow(`release.yml`).

Reply:
Moreover, this change was applied after discussion in this PR above: https://github.com/checkstyle/checkstyle/pull/12741#discussion_r1108528505